### ISSUE TITLE
XState\Test coverage reports covered states.

### ISSUE THREAD

**Issue by arturohernandez10** [statelyai/xstate#729]
**Description**

I think the app coverage is wrong in this case since it reports missing coverage. But those states were covered.

Here is the machine.
```js
  const feedbackMachine = Machine({
    id: 'feedback',
    initial: 'logon',
    states: {
      logon: {
        initial: 'empty',
        states: {
          empty: {
            on: {
              ENTER_LOGON: "filled"
            }
          },
          filled: { type: 'final' }
        },
        on: {
          LOGON_SUBMIT: 'ordermenu'
        },
        meta: { test: testInLogon }
      } ,
      ordermenu: {
        type: 'final',
        meta: { test: testInOrderMenu }
      }
    }
  });
```

And the output...

```
 RUNS  src/App.e2e.js

Test Suites: 0 of 1 total
Tests:       0 total
 FAIL  src/App.e2e.js (15.61s)
  feedback app
    × coverage (13ms)
    reaches state: {"logon":"empty"}
      √ via  (1459ms)
    reaches state: {"logon":"filled"}
      √ via ENTER_LOGON (1115ms)
    reaches state: "ordermenu"
      √ via ENTER_LOGON → LOGON_SUBMIT (1663ms)
      × via LOGON_SUBMIT (10017ms)

  ● feedback app › reaches state: "ordermenu"  › via LOGON_SUBMIT

    Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.

      70 |     describe(plan.description, () => {
      71 |       plan.paths.forEach((path, i) => {
    > 72 |         it(
         |         ^
      73 |           path.description,
      74 |           async () => {
      75 |             // eslint-disable-next-line no-undef

      at Spec (node_modules/jest-jasmine2/build/jasmine/Spec.js:85:20)
      at it (src/App.e2e.js:72:9)
          at Array.forEach (<anonymous>)
      at Suite.forEach (src/App.e2e.js:71:18)

  ● feedback app › coverage

    Missing coverage for state nodes:
        feedback.logon.empty
        feedback.logon.filled

      85 |
      86 |   it('coverage', () => {
    > 87 |     _testModel.testCoverage();
         |                ^
      88 |   });
      89 | });
      90 |

      at TestModel.Object.<anonymous>.TestModel.testCoverage (node_modules/@xstate/test/src/index.ts:300:13)
      at Object.testCoverage (src/App.e2e.js:87:16)

Test Suites: 1 failed, 1 total
Tests:       2 failed, 3 passed, 5 total
Snapshots:   0 total
Time:        16.458s
Ran all test suites related to changed files.

Watch Usage: Press w to show more.
```

**Expected Result**

I would expect no missing nodes in the coverage report.

**Actual Result**

Two states are shown as not covered.

**Reproduction**

If you need additional detail please ask me here...

**Comment by drmikecrowe** [comment]
@arturohernandez10 Quick question -- if you omit `type: 'final'` from the states, does the coverage work?  I wonder if this could be a duplicate of my #823

**Comment by davidkpiano** [comment]
Coverage is no longer included in `@xstate/test@alpha`; this will likely be a separate thing.

**Comment by viestat** [comment]
> Coverage is no longer included in `@xstate/test@alpha`; this will likely be a separate thing.

@davidkpiano I get that it is not included anymore, but it is in `0.5.1`. Does this mean that people using `0.5.1` should consider this is broken, it wont be fixed (in this package) and not to be used?

Because I have not found a way to make the `testCoverage()` method work. Not even from following the quick-start  example in [here](https://xstate.js.org/docs/packages/xstate-test/#quick-start)

**Comment by davidkpiano** [comment]
> > Coverage is no longer included in `@xstate/test@alpha`; this will likely be a separate thing.
> 
> @davidkpiano I get that it is not included anymore, but it is in `0.5.1`. Does this mean that people using `0.5.1` should consider this is broken, it wont be fixed (in this package) and not to be used?
> 
> Because I have not found a way to make the `testCoverage()` method work. Not even from following the quick-start  example in [here](https://xstate.js.org/docs/packages/xstate-test/#quick-start) 

Yes, the next major version will be a breaking change that does not include coverage. This might be added as a helper later, but won't be part of the test model.
